src/components/UserDetailsContainer.jsx

```diff
@@ -35,14 +38,37 @@ export class UserDetailsContainer extends Component {
 
   emptyUsersList = () => (isEmpty(this.props.users));
 
+  toggleFilter = () => {
+    this.setState(({ toggleOn }) => ({ toggleOn: !toggleOn }));
+  }
+
   render() {
     return (
       <NavbarComponent title="Users">
         <div className="users-list">
           <div id="page-heading-section">
             <Header as="h1" id="page-headings" floated="left" content="Users List" />
             <Divider id="assets-divider" />
+            <div
+              className={this.state.toggleOn ? 'clicked' : 'unclicked'}
+              id="filter-button"
+              onClick={this.toggleFilter}
+              role="presentation"
+            >
+              {this.state.toggleOn ?
+                <div id="close-filter">
+                  close X
+                </div> :
+                <div id="lines">
+                  <div className="burger-line" />
+                  <div className="burger-line" />
+                  <div className="burger-line" />
+                </div>
+              }
+              FILTERS
+            </div>
           </div>
+          {this.state.toggleOn ? <FilterComponent /> : null}
```

Comment:
A ternary operator that has `null` or `''`  as one of the possible values can be shorten by `&&`. E.g.
```
{this.state.toggleOn && <FilterComponent />
```